auth/auth.go

```diff
@@ -63,7 +64,7 @@ func setLoginToken(token string, w http.ResponseWriter) {
 // A user has these permission if they are in the super users list or if the list is empty,
 // in which case all users are super users.
 func IsSuperUser(superUsers []string, u User) bool {
-	if u == nil {
+	if u == nil || reflect.ValueOf(u).IsNil() {
```

Comment:
can we pass user as a pointer to avoid needing to use reflection?

Reply:
User is an interface, so changing this to take in a pointer to the interface will require callers to do something ugly to be able to pass in their user implementations. I think we want to avoid that

Reply:
Ugg. Lets add a method to User to check this, though what's the case where we have a non nil interface pointing to a nil value?